pynicotine/gtkgui/ui/popovers/wishlist.ui

```diff
@@ -77,13 +77,14 @@
                             <child>
                               <object class="GtkImage">
                                 <property name="visible">1</property>
-                                <property name="icon-name">edit-clear-symbolic</property>
+                                <property name="icon-name">list-remove-symbolic</property>
                               </object>
                             </child>
                             <child>
                               <object class="GtkLabel">
                                 <property name="visible">1</property>
-                                <property name="label" translatable="yes">_Remove</property>
+                                <property name="label" translatable="yes">R_emove Wish</property>
+                                <property name="tooltip-text">Shift+Delete</property>
```

Comment:
- Fixed: Resolved mnemonic conflict with _Result Filters that meant shortcut had to be pressed twice to Remove item.
- Added: ToolTip for Shift+Delete shortcut.